README.md

```diff
@@ -69,6 +69,39 @@ cat $(find . -type f -name "*.log" | sort -n | tail -1) | jq -cs '.'
 
 For some useful queries on the log data, see [📚 Useful queries on crawler logs](https://github.com/gt-scheduler/crawler/wiki/%F0%9F%93%9A-Useful-queries-on-crawler-logs).
 
+### Using the Python Finals Data Scraper
+
+Install the necessary Python modules with the included `requirements.txt` file
```

Comment:
At the risk of being overly verbose, I'd like to also include the required min version of Python. (I based this on the GitHub Actions config, let me know if this is wrong)

```suggestion
First, ensure [Python 3.9 or newer](https://www.python.org/downloads/) is installed. Then, install the necessary Python modules with the included `requirements.txt` file:
```